libraries/src/CMS/Mvc/Factory/MvcFactory.php

```diff
@@ -62,6 +62,11 @@ public function __construct($namespace, CMSApplicationInterface $application)
 	 */
 	public function createModel($name, $prefix = '', array $config = array())
 	{
+		if (empty($prefix) && $this->application->isClient('api'))
```

Comment:
Dirty hack to ensure that models are correctly loaded. We should use our own ApiFactory